app/assets/javascripts/discourse/app/components/table-header-toggle-bulk.js

```diff
@@ -0,0 +1,26 @@
+import TableHeaderToggleComponent from "discourse/components/table-header-toggle";
+
+export default TableHeaderToggleComponent.extend({
+  click(e) {
+    const self = this;
```

Comment:
You shouldn't need to use a `self` variable here. If you make `onClick = (sel, callback) => {` then its `this` will be the parent `this`.